[Tabs] Update MDCItemBarCell edge insets to 16.0f, per the material spec.

mrhappyasthma opened the pull request:
The spec was updated to use 16.0f as the edge padding for tabs, regardless of size class: https://i.imgur.com/0CT0N1h.png

https://material.io/go/design-tabs#spec

Closes #5687

material-automation commented on components/Tabs/src/private/MDCItemBarCell.m:
```diff
@@ -36,6 +36,10 @@
 
+/// Outer edge padding from spec: https://material.io/go/design-tabs#spec.
+static const UIEdgeInsets kEdgeInsets = { .top = 0.0f, .right = 16.0f,
+                                          .bottom = 0.0f, .left = 16.0f };
```
clang-format suggested the following change:


```
-static const UIEdgeInsets kEdgeInsets = { .top = 0.0f, .right = 16.0f,
-                                          .bottom = 0.0f, .left = 16.0f };
+static const UIEdgeInsets kEdgeInsets = {
+    .top = 0.0f, .right = 16.0f, .bottom = 0.0f, .left = 16.0f};
```

<!-- identifier: clang-format -->
<!-- File: components/Tabs/src/private/MDCItemBarCell.m -->

material-automation commented:
clang-format recommended changes.

Consider installing [git-clang-format](https://github.com/llvm-mirror/clang/blob/master/tools/clang-format/git-clang-format) and running the following command:

```bash
git clang-format $(git merge-base origin/develop HEAD)
```

<!-- identifier: clang-format -->

mrhappyasthma committed:
Update MDCItemBarCell edge insets to 16.0f, per the material spec.

The spec was updated to use 16.0f as the edge padding for tabs, regardless of size class: https://i.imgur.com/0CT0N1h.png

https://material.io/go/design-tabs#spec

mrhappyasthma commented:
Commit originally had the wrong author due to cloning a new repo. Amended the commit. CLA should be resolved now.

mrhappyasthma commented:
Clang format issue should be fixed.

mrhappyasthma commented:
Closes #5687

mrhappyasthma committed:
Merge branch 'develop' into spec

romoore commented:
Thanks for the PR, @mrhappyasthma . I'm running some internal tests for this change. 

Googlers, check: cl/221315994

romoore commented:
Some internal screenshot tests have failed and now show the updated (wider) spacing. This is expected, and it looks like only a handful.

romoore commented on components/Tabs/src/private/MDCItemBarCell.m:
```diff
@@ -36,6 +36,10 @@
 
+/// Outer edge padding from spec: https://material.io/go/design-tabs#spec.
+static const UIEdgeInsets kEdgeInsets = {
+    .top = 0.0f, .right = 16.0f, .bottom = 0.0f, .left = 16.0f};
```
Nit: Please use `16` instead of `16.0f`. We recently change our style across the entire project to stop using the `0.1f` format for CGFloat literals.

mrhappyasthma replied:
Should be updated now to use integer constants. PTAL.

romoore reviewed (approved):
Small nit, otherwise looks good.

mrhappyasthma committed:
Update MDCItemBarCell.m

Drop the .f notation on float constants.

material-automation commented on components/Tabs/src/private/MDCItemBarCell.m:
```diff
@@ -36,6 +36,10 @@
 
+/// Outer edge padding from spec: https://material.io/go/design-tabs#spec.
+static const UIEdgeInsets kEdgeInsets = {
+    .top = 0, .right = 16, .bottom = 0, .left = 16};
```
clang-format suggested the following change:


```
-static const UIEdgeInsets kEdgeInsets = {
-    .top = 0, .right = 16, .bottom = 0, .left = 16};
+static const UIEdgeInsets kEdgeInsets = {.top = 0, .right = 16, .bottom = 0, .left = 16};
```

<!-- identifier: clang-format -->
<!-- File: components/Tabs/src/private/MDCItemBarCell.m -->

material-automation commented:
clang-format recommended changes.

Consider installing [git-clang-format](https://github.com/llvm-mirror/clang/blob/master/tools/clang-format/git-clang-format) and running the following command:

```bash
git clang-format $(git merge-base origin/develop HEAD)
```

<!-- identifier: clang-format -->

mrhappyasthma committed:
Update MDCItemBarCell.m

Clang format recommended change.

mrhappyasthma commented:
Un-linewrapped the line now to satisfy clang-format check (hopefully).
